java/basic/README.md

```diff
@@ -0,0 +1,13 @@
+# Sentry Basic Example
+
+This project can be compiled and run with [Maven][maven]:
+
+    mvn compile exec:java
+
+To report to an existing Sentry installation, set the `SENTRY_DSN` environment
+variable:
+
+    SENTRY_DSN=https://public:private@host:port/1 mvn exec:java
+
+[maven]: http://maven.apache.org/
+[sentry-java]: https://github.com/getsentry/sentry-java
```

Comment:
Looks like we didn't make use of this `sentry-java` reference here (or in the other examples). We should probably add a line somewhere linking people to the project or [documentation](https://docs.sentry.io/clients/java/) incase they land here and want more.